src/App.css

```diff
@@ -112,7 +113,7 @@ body,
 }
 
 .page-content-child {
-    width: 90%;
+    width: 100%;
```

Comment:
change this to 60% for desktop css and 100% for mobile version.

Reply:
Applie 100% to max-width 768px. Tell me if you think it's best for a different breakpoint.